client/my-sites/checkout/post-checkout-upsell-experiment-redirector/index.tsx

```diff
@@ -43,8 +43,10 @@ export default function PostCheckoutUpsellExperimentRedirector( {
 			return;
 		}
 
-		if ( upsellExperimentAssignmentName === experimentAssignment?.variationName ) {
+		if ( true ) {
+		// if ( upsellExperimentAssignmentName === experimentAssignment?.variationName ) {
```

Comment:
This is a temporary change, as I didn't want to go through the whole process of documenting and creating a new experiment.